identity-service/package.json

```diff
@@ -16,7 +16,7 @@
     "lint-fix": "./node_modules/.bin/standard --fix"
   },
   "dependencies": {
-    "@audius/libs": "^1.1.9",
+    "@audius/libs": "^1.1.16",
```

Comment:
should we explicitly set versions like we do in other services?

Reply:
I think we should - updating

Reply:
bumped